### PR TITLE
Update finding submission form

### PR DESCRIPTION
**Pull request by captainmangoC4**
- Add lines of code input fields
- Remove non-critical risk category

### CODE REVIEW

File: src/components/reporter/Form.js [hunk 1]
```diff
@@ -3,79 +3,82 @@ import { StaticQuery, graphql } from "gatsby";
...
+  helpText: "Used to send a copy of this form for your records",
+  widget: "text",
+  required: true,
+};
+
+const addressField = {
+  name: "address",
+  label: "Polygon address",
+  helpText:
+    "Address where your prize should go to (and retroactive token reward should C4 be tokenized later). If you use a smart contract wallet, please contact one of our organizers in Discord in addition to adding the address here.",
```
Comment (sockdrawermoney) [inline src/components/reporter/Form.js:27]: `(and retroactive token reward should C4 be tokenized later)` isn't necessary anymore :)

File: src/components/reporter/Form.js [hunk 2]
```diff
@@ -3,79 +3,82 @@ import { StaticQuery, graphql } from "gatsby";
...
     },
     {
-      name: "title",
-      label: "Title",
-      helpText:
-        "Summarize your findings for the bug or vulnerability. (This will be the issue title.)",
-      widget: "text",
-      required: true,
+      label: "1 — QA (Low and Non-critical)",
+      value: "1 (Low Risk)",
```
Comment (sockdrawermoney) [inline src/components/reporter/Form.js:53]: Let's change the label to `QA Report (low / non-critical)` and the value to `QA Report`

File: src/components/reporter/Form.js [hunk 3]
```diff
@@ -3,79 +3,82 @@ import { StaticQuery, graphql } from "gatsby";
...
-          label: "2 — Medium Risk",
-          value: "2 (Med Risk)",
-        },
-        {
-          label: "3 — High Risk",
-          value: "3 (High Risk)",
-        },
-      ],
+      label: "2 — Medium Risk",
+      value: "2 (Med Risk)",
```
Comment (sockdrawermoney) [inline src/components/reporter/Form.js:57]: We can just drop the numbers from the values I think and just have the label and value both as "Medium Risk"

File: src/components/reporter/Form.js [hunk 4]
```diff
@@ -3,79 +3,82 @@ import { StaticQuery, graphql } from "gatsby";
...
     },
     {
-      name: "details",
-      label: "Vulnerability details",
-      helpText: "Link to all referenced sections of code in GitHub",
-      widget: "textarea",
-      required: true,
+      label: "3 — High Risk",
```
Comment (sockdrawermoney) [inline src/components/reporter/Form.js:60]: We can just drop the numbers from the values I think and just have the label and value both as "High Risk"

**Review by DianaPerkinsDesign — APPROVED** [review]
LGTM!

File: src/components/reporter/Form.js [hunk 5]
```diff
@@ -3,79 +3,82 @@ import { StaticQuery, graphql } from "gatsby";
...
+  required: true,
+  options: [
     {
-      name: "address",
-      label: "Polygon address",
-      helpText:
-        "Address where your prize should go to (and retroactive token reward should C4 be tokenized later). If you use a smart contract wallet, please contact one of our organizers in Discord in addition to adding the address here.",
-      widget: "text",
-      required: true,
+      label: "0 — Gas Optimization",
```
Comment (sockdrawermoney) [inline src/components/reporter/Form.js:48]: We can just drop the numbers from the values I think and just have the label and value both as "Gas Optimizations"

**Review by sockdrawermoney — APPROVED** [review]
approved with these minor tweaks

**Pull request merged**